static/js/task/controllers/task.controller.js

```diff
@@ -22,17 +22,22 @@
             self.task_worker_id = $routeParams.taskWorkerId;
             self.task_id = $routeParams.taskId;
 
-            if (!self.task_worker_id) { //if they navigate away mid-queue and then attempt task from task feed
```

Comment:
sorry if this was unclear. to replicate: start the queue with some tasks in it, midway navigate to the task feed, attempt a task. i get this error:
<img width="1280" alt="screen shot 2015-09-06 at 3 31 15 pm" src="https://cloud.githubusercontent.com/assets/4563883/9706197/929c66fa-54ac-11e5-8b85-1c5bec4ee513.png">

message me on slack if this is still unclear.